README.md

```diff
@@ -1,8 +1,19 @@
-# identity-framework
-===============================
-
+# carbon-identity-framework
 ---
 
+This repository contains the core components and features required by the [WSO2 Identity Server](https://github.com/wso2/product-is) product.
+ 
+
+## Building from the source
+
+If you want to build **carbon-identity-framework** from the source code:
+
+1. Install Java7 or above
+1. Install Apache Maven 3.x.x(https://maven.apache.org/download.cgi#)
+1. Get a clone or download the source from this repository (https://github.com/wso2/carbon-identity-framework.git)
+1. Run the Maven command ``mvn clean install`` from the ``carbon-identity-framework`` directory.
+
+
 |  Branch | Build Status |
 | :------------ |:-------------
-| master      | [![Build Status](https://wso2.org/jenkins/job/carbon-identity-framework/badge/icon)](https://wso2.org/jenkins/job/carbon-identity-framework) |
+| master      | [![Build Status](https://wso2.org/jenkins/job/platform-builds/job/carbon-identity-framework/badge/icon)](https://wso2.org/jenkins/job/platform-builds/job/carbon-identity-framework/) |
```

Comment:
@madurangasiriwardena The Jenkins badge should go just after the title, not the bottom of the README.